### PR TITLE
feature: add morning afternoon/evening dzikir screen

### PR DESCRIPTION
**Pull request by fahmialfareza**
- Refactor some code
- Add morning afternoon/evening dzikir screen
- Add some components
- Add async storage getData and storeData

### CODE REVIEW

File: components/dzikirDua/DzikirDuaItem.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  arabic: string;
+  arabic_latin: string;
+  meaning_id: string;
+  state: HeaderDzikirDuaState;
+}
+
+const DzikirDuaItem = ({
+  note,
+  arabic,
+  arabic_latin,
```
Comment (Afatkharrofiqi) [inline components/dzikirDua/DzikirDuaItem.tsx:25]: gimana kalau di buat camelCase? standarin aja semua param atau variable jadi camelCase

Comment (fahmialfareza) [reply components/dzikirDua/DzikirDuaItem.tsx:25]: Biasanya kalo di TS interface, enum, class itu pake PascalCase sih bro

Comment (Afatkharrofiqi) [reply components/dzikirDua/DzikirDuaItem.tsx:25]: ouh oke, ngikut aja belum pegang TS wkwk, agak triggered sebenere wkwkw

Comment (fahmialfareza) [reply components/dzikirDua/DzikirDuaItem.tsx:25]: Wkwk syap
Kenapa biasa pake PascalCase wkwk itu biar bisa bedain mana modelnya sama variable nya
Mirip2 di Golang lah wkwk

File: components/dzikirDua/DzikirDuaItem.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  Platform,
+} from 'react-native';
+import RenderHtml from 'react-native-render-html';
+
+import { HeaderDzikirDuaState } from './HeaderDzikirDuaOptions';
+
+interface DzikirDuaItemProps {
+  note: string;
+  arabic: string;
+  arabic_latin: string;
```
Comment (Afatkharrofiqi) [inline components/dzikirDua/DzikirDuaItem.tsx:17]: eh btw nemu iki https://google.github.io/styleguide/tsguide.html#interfaces-vs-type-aliases, ko pakai camelCase ya?

Comment (Afatkharrofiqi) [reply components/dzikirDua/DzikirDuaItem.tsx:17]: bukan nama interfacenya tapi propertinya

Comment (Afatkharrofiqi) [reply components/dzikirDua/DzikirDuaItem.tsx:17]: arabic_latin -> arabicLatin

Comment (fahmialfareza) [reply components/dzikirDua/DzikirDuaItem.tsx:17]: Oalah, bener2 tak buat camelCase besok ya haha 🤣

**Comment by fahmialfareza** [comment]
@Afatkharrofiqi cek cuy udah tak updet haha

**Review by santoadji21 — APPROVED** [review]

**Pull request merged**